WeatherForecast/WeatherForecast/ImageManager.swift

```diff
@@ -8,15 +8,15 @@
 import UIKit
 
 protocol ImageManagerProtocol {
-    func fetchImage(of iconName: String, completion: @escaping (UIImage?) -> ())
+    func fetchImage(of iconName: String, completion: @escaping (UIImage?) -> Void)
     var imageChache: NSCache<NSString, UIImage> { get }
```

Comment:
```suggestion
    var imageCache: NSCache<NSString, UIImage> { get }
```
Xcode 메뉴바에서 Edit > Format > Spelling and Grammer > Check Spelling While Typing 옵션을 활성화하시면 캡처의 빨간 박스처럼 틀린 철자를 확인하실 수 있습니다.
<img width="841" alt="image" src="https://github.com/yagom-academy/refactoring-weather/assets/69730931/841faf67-6260-4388-85ac-5d8d5dbd93c4">